### DISCUSSION TITLE
Haggis is a national dish of which country?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#777] [Galaxy Trivia] [answered]
**Bonus:** How many people does 1 haggis feed?

**Comment by SANKARAMDAS** [comment]
Scotland
haggis, the national dish of Scotland, a type of pudding composed of the liver, heart, and lungs of a sheep (or other animal), minced and mixed with beef or mutton suet and oatmeal and seasoned with onion, cayenne pepper, and other spices.

Award Winning Traditional Haggis
Sizes range from the 454g haggis which feed two to three people to the chieftain haggis that feeds 16.

**Comment by peterbaccaro** [comment, marked as answer]
Haggis is a traditional national dish of **Scotland**. It is a savory pudding made from sheep's offal (heart, liver, and lungs), minced with onions, suet, spices, and oatmeal. The mixture is then traditionally encased in a sheep's stomach and simmered for several hours. Haggis is often served with neeps (turnips) and tatties (potatoes) and is commonly associated with Scottish cuisine and cultural celebrations such as Burns Night, which commemorates the poet Robert Burns.

The serving size of haggis can vary depending on various factors such as the size of the haggis, the accompanying dishes, and individual appetite. Generally, **a standard haggis weighing around 1.5 to 2 pounds (680 to 907 grams) is estimated to serve approximately 4 to 6 people as a main course**. However, this can be adjusted based on portion sizes and the number of accompanying side dishes. It's always a good idea to consider the specific recipe and consult with the person preparing the meal for a more accurate estimate.
